[CI] DocsClientYamlTestSuiteIT.test {yaml=reference/rollup/apis/rollup-api/line_13}

<!--
Please fill out the following information, and ensure you have attempted
to reproduce locally
-->

**Build scan**:

https://gradle-enterprise.elastic.co/s/djhhfvapry6co

**Repro line**:

(note that this contains `-Dbuild.snapshot=false`)

```
./gradlew ':docs:integTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/rollup/apis/rollup-api/line_13}" -Dtests.seed=5B9E56C11F05143F -Dtests.security.manager=true -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=fi-FI -Dtests.timezone=Australia/Currie -Druntime.java=11
```

**Reproduces locally?**:

Don't know, I couldn't get the test to run 🤷‍♂️ 

**Applicable branches**:

`master`

**Failure history**:
<!--
Link to build stats and possible indication of when this started failing and how often it fails
<https://build-stats.elastic.co/app/kibana>
-->
**Failure excerpt**:

```
org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT > test {yaml=reference/rollup/apis/rollup-api/line_13} FAILED
    java.lang.AssertionError: Failure at [reference/rollup/apis/rollup-api:51]: expected [2xx] status code but api [raw[method=POST path=my-index-000001/_rollup]] returned [400 Bad Request] [{"error":"no handler found for uri [/my-index-000001/_rollup?error_trace=true] and method [POST]"}]
        at __randomizedtesting.SeedInfo.seed([5B9E56C11F05143F:D3CA691BB1F979C7]:0)
        at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:414)
        Caused by:
        java.lang.AssertionError: expected [2xx] status code but api [raw[method=POST path=my-index-000001/_rollup]] returned [400 Bad Request] [{"error":"no handler found for uri [/my-index-000001/_rollup?error_trace=true] and method [POST]"}]
            at org.elasticsearch.test.rest.yaml.section.DoSection.execute(DoSection.java:296)
            at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:407)
```

--- a/docs/build.gradle
+++ b/docs/build.gradle
@@ -52,13 +52,11 @@ testClusters.matching { it.name == "integTest"}.configureEach {
   if (singleNode().testDistribution == DEFAULT) {
     setting 'xpack.license.self_generated.type', 'trial'
     setting 'indices.lifecycle.history_index_enabled', 'false'
-    if (BuildParams.isSnapshotBuild() == true) {
-      systemProperty 'es.rollup_v2_feature_flag_enabled', 'true'
-    }
     if (BuildParams.isSnapshotBuild() == false) {
       systemProperty 'es.autoscaling_feature_flag_registered', 'true'
     }
     setting 'xpack.autoscaling.enabled', 'true'
+    systemProperty 'es.rollup_v2_feature_flag_enabled', 'true'
     keystorePassword 's3cr3t'
   }
 

--- a/docs/reference/rollup/apis/rollup-api.asciidoc
+++ b/docs/reference/rollup/apis/rollup-api.asciidoc
@@ -38,8 +38,7 @@ POST /my-index-000001/_rollup
   ]
 }
 ----
-// CONSOLE
-// TEST[skip:"AwaitsFix https://github.com/elastic/elasticsearch/issues/65544"]
+// TEST[setup:my_index]
 
 
 [[rollup-api-request]]

